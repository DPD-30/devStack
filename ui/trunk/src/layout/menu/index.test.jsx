import React from 'react'
import { renderToString } from 'react-dom/server'

import LayoutMenuIndex from './index.jsx'

jest.unmock('./index.jsx')


describe('', () => {
    it('LayoutMenuIndex should match snapshot', () => {
        const comp = renderToString(<LayoutMenuIndex />)

        expect(comp).toMatchSnapshot()
    })
})
