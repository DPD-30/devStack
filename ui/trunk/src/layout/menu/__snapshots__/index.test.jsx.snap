// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMenuIndex should match snapshot 1`] = `
<nav class="navbar navbar-expand-md bg-dark"
     data-reactroot
>
  <button aria-label="Toggle navigation"
          type="button"
          class="mr-2 navbar-toggler"
  >
    <span aria-hidden="true"
          class="fa fa-bars"
    >
    </span>
  </button>
  <div class="collapse navbar-collapse">
    <ul class="navbar-nav">
    </ul>
  </div>
</nav>
`;
